src/main/kotlin/Locations.kt

```diff
@@ -33,4 +33,14 @@ class Locations {
 
     @Location("/teams/{uuid}")
     data class Team(val uuid: String = "")
+
+    @Location("/message")
```

Comment:
assume u didnt read over the article you went over, but i made a typo wheni said it shou
ld me message since REST requirements are that they are plural so it should be messages
example:

GET /tickets - Retrieves a list of tickets
GET /tickets/12 - Retrieves a specific ticket
POST /tickets - Creates a new ticket
PUT /tickets/12 - Updates ticket #12
PATCH /tickets/12 - Partially updates ticket #12
DELETE /tickets/12 - Deletes ticket #12